Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift

```diff
@@ -222,6 +222,11 @@ final class SchemeDescriptorsGenerator: SchemeDescriptorsGenerating {
         var preActions: [XCScheme.ExecutionAction] = []
         var postActions: [XCScheme.ExecutionAction] = []
 
+        let testPlans: [XCScheme.TestPlanReference]? = testAction.testPlans?.map {
```

Comment:
I'd recommend extending the [SchemeLinter](https://github.com/tuist/tuist/blob/main/Sources/TuistGenerator/Linter/SchemeLinter.swift) to make sure that all the provided test plans exist. Otherwise, we might end up generating a scheme with broken references to test plans.

Reply:
added a warning